modules/usbip.nix

```diff
@@ -2,7 +2,7 @@
 
 let
   usbipd-win-auto-attach = pkgs.fetchurl {
-    url = "https://raw.githubusercontent.com/dorssel/usbipd-win/v3.1.0/Usbipd/wsl-scripts/auto-attach.sh";
+    url = "https://raw.githubusercontent.com/dorssel/usbipd-win/v4.2.0/Usbipd/WSL/auto-attach.sh";
     hash = "sha256-KJ0tEuY+hDJbBQtJj8nSNk17FHqdpDWTpy9/DLqUFaM=";
```

Comment:
That hash definitely changed with https://github.com/dorssel/usbipd-win/commit/b37ff8dfeebe17b45c34ae0a8b1db0c629826427#diff-d4cc2627d14099db7791bdfd4d9af17d019ce446ea51ac3cd2d05d291231ff0d

Reply:
Should we just pull the repo as a flake input?

Reply:
That likely changes very often and we don't really need it.